### DISCUSSION TITLE
different AR modes? types?

### DISCUSSION THREAD

**Discussion by iiLearner** [google/model-viewer#2165] [Q&A]
### Description

Hello! I'm using model-viewer for my 3D objects on my web page. 
I have noticed when the user goes on model-viewer page, if i immediately send the user into AR through simulating a button click, the AR is perfect, with most of features such as photo taking, prompt to help user place the object and such. However, if the user has to manually click the 'go into ar' button manually after even a few seconds, the AR experience is different. There's no prompts and its honestly just not good at all. The code is the same, everything is the same except the fact in one case the user is sent it AR immediately and in the other case user has to click the button.

This is only for Android.
![image](https://user-images.githubusercontent.com/24250663/113008140-11c21580-9177-11eb-912b-76c7805c0139.png)
![image](https://user-images.githubusercontent.com/24250663/113008153-1686c980-9177-11eb-973d-d150d47d8a00.png)

### Version

- model-viewer: 1.6.0

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x] WebXR
- [x] SceneViewer
- [x] QuickLook

**Comment by pzyu** [comment]
The first screenshot looks like WebXR, and the second looks like SceneViewer. 

Maybe try limiting the ar modes to to SceneViewer only? See: https://modelviewer.dev/examples/augmentedreality/index.html#sceneViewer

Prompts can also be added in WebXR: https://modelviewer.dev/examples/augmentedreality/#webXR

**Reply by AndresSalomon1990** [reply]
Agree with @pzyu . You have to change the default ar-modes attribute to choose first the scene-viewer instead of webxr (for iOS it always go to quick-look) like this:

`ar-modes="scene-viewer webxr quick-look"`

Be aware of this open issue (which is also happening to me in my phone): https://github.com/google/model-viewer/issues/2264
